lms/templates/learner_dashboard/upgrade_message_2017.underscore

```diff
@@ -4,7 +4,7 @@
     <span class="price">  <%- price %></span>
 </div>
 <div class="action col-12 md-col-4">
-    <button href="<%- upgrade_url %>" type="button" class="btn-brand btn cta-primary upgrade-button">
+    <a href="<%- upgrade_url %>" class="btn-brand btn cta-primary upgrade-button">
```

Comment:
but this item doesn't take any actions and is just a link. Shouldn't it be a link from an a11y perspective?
(it is a link in the current view, i think i incorrectly changed it to a button)